include/vcpkg/base/cofffilereader.h

```diff
@@ -324,6 +324,8 @@ namespace vcpkg
         THUMB = 0x1c2,         // Thumb
         WCEMIPSV2 = 0x169,     // MIPS little-endian WCE v2
         LLVM_BITCODE = 0x4342, // LLVM bitcode https://www.llvm.org/docs/BitCodeFormat.html#llvm-ir-magic-number
+        LOONGARCH32 = 0x6232,  // LoongArch32
```

Comment:
Confirmed: these values are in the current PE/COFF spec. https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#machine-types

Reply:
Hello BillyONeal, I'm not a programmer, I just started not long ago and may be inexperienced, if you have any better suggestions please don't hesitate to ask!
    are you saying that for Linux you don't need to specify it in the coffilereader?
    I've re-updated the comments and refined the [LoongArch-specific C/C++ Built-in Macros(Table 14/15)](https://loongson.github.io/LoongArch-Documentation/LoongArch-toolchain-conventions-EN.html)

Reply:
>are you saying that for Linux you don't need to specify it in the coffilereader?

No, I was saying for other reviewers that these values match what the spec says, not asking you to change anything.